src/assets/scripts/client/constants/themes/default/modules/scope.js

```diff
@@ -124,6 +124,14 @@ export const SCOPE_THEME = {
      */
     SID: COLOR.BLUE_VERY_LIGHT,
 
+    /**
+     * Color of the lines drawn between fixes on STARs
+     *
+     * @memberof SCOPE_THEME
+     * @property STAR
+     */
+    STAR: COLOR.BLUE_VERY_LIGHT,
```

Comment:
How about `COLOR.RED` like the arrivals? Then, if both SIDs and STARs are shown, they can be distinguished (even if it's probably going to be just a hot mess of lines all over the place :wink:).

Reply:
done